src/utils/theme/index.js

```diff
@@ -46,21 +57,21 @@ export const COLORS = {
   signed_primary: '#0391FB',
   signed_primary_15: '#0391FB26',
   signed_secondary: '#275D8A',
-  default_signed_secondary: '#69A2F2',
+  default_signed_secondary: '#0391FB',
   // anon_primary: '#107793',
   // anon_primary_15: '#10779326',
   // anon_secondary: '#3A9BB6',
-  anon_primary: '#00C4C6',
-  anon_primary_15: '#00C4C626',
+  anon_primary: '#00ABB2',
+  anon_primary_15: '#00ABB226',
   anon_secondary: '#184A57',
   radical_red: '#FF3466',
   light_silver: '#D2D4DB',
   balance_gray: '#9B9FA9',
   gray: '#E8EBED',
-  lightgrey: '#F5F6F7',
+  lightgrey: '#233343', // changed to gray 100 #F5F6F7
   redalert: '#FF2E63',
-  blackgrey: '#828282',
-  greySubtile1: '#9B9FA9',
+  blackgrey: '#8C939F', // changed to gray 400
+  greySubtile1: '#8C939F', // changed to gray 400
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Several color changes have been made here. As with the previous comments, ensure that these changes do not negatively impact the readability and overall look of the user interface. Also, the color `lightgrey` has been changed to a dark blue-gray which could lead to confusion. Consider renaming this color to something more descriptive of its actual color.
```diff
-  lightgrey: '#233343', // changed to gray 100 #F5F6F7
+  darkBlueGray: '#233343',
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->